src/applications/disability-benefits/781/pages/ptsdChoice.js

```diff
@@ -1,70 +1,34 @@
-import React from 'react';
-import {
-  ptsdNameTitle,
-} from '../helpers';
-import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
+import { ptsdNameTitle781, uploadPtsdDescription781, ptsdChoiceDescription } from '../helpers';
 
-const uploadPtsdDescription = () => {
-  return (
-    <div>
-      <p>
-        The following questions will help us understand more about your
-        [PTSDclassification]-related PTSD. None of the questions we‘ll ask you are
-        required, but any information you provide here will help us research your claim.
-      </p>
-      <p>
-        If you have already completed a Claim for Service Connection for
-        Post-Traumatic Stress Disorder (VA Form 21-0781), you can upload it here
-        instead of answering the questions about your PTSD.
-      </p>
-      <p>How would you like to provide information about your PTSD?</p>
-    </div>
-  );
-};
-
-const ptsdChoiceDescription = (
```

Comment:
Same here as above.

Reply:
@igreulich  suggested that all helpers be kept in helpers. So I moved it to helpers. You guys let me know what's best, and I will do it. Thanks! @robfelder